browser/ui/tabs/shared_pinned_tab_service.cc

```diff
@@ -721,7 +722,24 @@ void SharedPinnedTabService::SynchronizeMovedPinnedTab(int from, int to) {
   }
 }
 
+void SharedPinnedTabService::TabDraggingEnded(Browser* browser) {
+  if (!in_tab_dragging_browsers_.count(browser)) {
+    return;
+  }
+
+  in_tab_dragging_browsers_.erase(browser);
```

Comment:
you can simply call `erase` (and don't call `count` above) and check it's return value to see if anything was erased

Reply:
Nice catch! Done.